### PR TITLE
Prep release

### PR DESCRIPTION
**Pull request by antarcticrainforest**
This prepares the, hopfully, last release of this software,

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,6 +33,8 @@ prepdocs:
 		--env EVALUATION_SYSTEM_CONFIG_FILE $(EVALUATION_SYSTEM_CONFIG_FILE) \
 		--env EVALUATION_SYSTEM_PLUGINS $(EVALUATION_SYSTEM_PLUGINS)
 	python3 -m bash_kernel.install
+	rm -rf docs/source/APIRef.rst
+	curl -Ls -o docs/source/APIRef.rst https://raw.githubusercontent.com/FREVA-CLINT/freva-nextgen/main/docs/source/databrowser/APIRef.rst
```
Comment (tropicrainforest) [inline Makefile]: I wanted to ask why we are using the databrowserAPI API reference documents here. IMHO having the databrowserAPI reference, which is included here but is not applicable in Freva, next to the plugin API (which has the `API Reference` header in the Freva documents) might confuse users/admins.  I think after deploying the docs, we have `API Reference` and `Databrowser Rest API` on the header.  just as a minor suggestion.

File: docs/source/whats-new.rst
Comment (tropicrainforest) [inline docs/source/whats-new.rst:1]: what r ur thoughts to update https://github.com/FREVA-CLINT/freva/blob/main/CHANGELOG.md as well, or completely get rid of it in the main dir of repo?

Comment (antarcticrainforest) [reply docs/source/whats-new.rst:1]: good point, I'll delete it then.

File: docs/source/whats-new.rst [hunk 2]
```diff
@@ -13,7 +13,9 @@ Bug fixes
 
 - Fixed plugin batchmode submit bug. Batchmode plugins did not pick up the
   correct evaluation system config file. This has been fixed now.
-
+- Corrected minor typos in documentation and code comments. (fix typo, minor typo, minor typo separator)
+- Temporarily disabled `test_plugin_status` unit-test in `plugin_command_test.py` for further investigation.
```
Comment (tropicrainforest) [inline docs/source/whats-new.rst]: this one has to be removed i think

File: docs/source/whats-new.rst [hunk 3]
```diff
@@ -13,7 +13,9 @@ Bug fixes
 
 - Fixed plugin batchmode submit bug. Batchmode plugins did not pick up the
   correct evaluation system config file. This has been fixed now.
-
+- Corrected minor typos in documentation and code comments. (fix typo, minor typo, minor typo separator)
```
Comment (tropicrainforest) [inline docs/source/whats-new.rst]: Would be better to change it to: Revised minor typographical errors in the documentation and code comments.

Comment (antarcticrainforest) [reply docs/source/whats-new.rst]: Ok

**Comment by antarcticrainforest** [comment]
Any more concerns?

**Comment by tropicrainforest** [comment]
no actually, LGTM

**Review by tropicrainforest — APPROVED** [review]

**Comment by antarcticrainforest** [comment]
Thanks!

**Pull request merged**